.github/workflows/updateIndex.yml

```diff
@@ -0,0 +1,53 @@
+
+name: Update Devices Index
+
+on:
+  pull_request:
+    types: [opened, edited]
+  issue_comment:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Setup repo
+        run: |
+          npm install
+          npm run prerelease
+
+      - name: Update index
+        id: check_index
+        run: |
+          npm run config -- -i
+          indexPath="packages/config/config/devices/index.json"
+          changedFiles=$(git diff-index --name-only HEAD)
+          if [ "$changedFiles" = "$indexPath" ]; then
+            echo "Index has changed"
+            git add $indexPath
+            echo "::set-output name=PUSH::true"
+          fi
+      - name: Comment a pull_request
+        if: steps.check_index.outputs.PUSH == 'true' && github.event_name == 'pull_request'
+        uses: mb2dev/github-action-comment-pull-request@1.0.0
+        with:
+          message: "The index file isn't updated. Comment 'update index' to automatically update it"
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Commit files
+        if: steps.check_index.outputs.PUSH == 'true' && github.event_name != 'pull_request' && github.event.comment.body == 'update index'
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git commit -m "fix(config): updated devices index.json"
+      - name: Push changes
+        if: steps.check_index.outputs.PUSH == 'true' && github.event_name != 'pull_request' && github.event.comment.body == 'update index'
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch: ${{ github.ref }}
```

Comment:
I think we don't need an action if we can just do `git push`:
https://github.community/t/how-does-one-commit-from-an-action/16127/7